ui/docs/manual/using/integration/frontend.md

```diff
@@ -88,112 +66,108 @@ const status = document.getElementById('status');
 </script>
 ```
 
-Now we are going to create an instance of a `WebAuth` from auth0. This is the
-entity that lets us communicate with the the auth0 service. This `WebAuth`
-instance will take some properties based on how your auth0 client is configured.
+Loading our page up to this point you will see our action link, and clicking
+it will navigate you to the familiar Taskcluster user interface to sign in.
+The URL takes the following query parameters:
 
-```js
-const link = document.getElementById('action-link');
-const status = document.getElementById('status');
-const auth = new auth0.WebAuth({
-  domain: 'auth.mozilla.auth0.com',
-  responseType: 'token id_token',
-  scope: 'taskcluster-credentials openid profile',
-  clientID: '<YOUR AUTH0 CLIENT ID>',
-  audience: '<YOUR AUTH0 CLIENT AUDIENCE>',
-  // This redirect URI is configured with your auth0 client.
-  // For this example, we will pretend that our current HTML
-  // page is going to be served no matter what route is used
-  redirectUri: '<YOUR AUTH0 CLIENT REDIRECT URL>'
-});
-```
+| Parameter | Required | Description |
+| --- | --- | --- |
+| response_type |✓ |The value MUST be "code" for requesting an authorization code, or "token” for requesting an access token. |
+| client_id | ✓ | The registered OAuth client. |
+| redirect_uri | ✓ | The URL to which the browser should be redirected after authorization. |
+| scope | ✓ | | An array of URIs to which the server is allowed to redirect the user. |
+| state | Optional (strongly recommended) | An opaque value used by the client to maintain state between the request and callback.  The authorization server includes this value when redirecting the user-agent back to the client.  The parameter SHOULD be used for preventing cross-site request forgery. |
+| expires | Optional | The requested lifetime of the resulting Taskcluster credentials, in a format understood by [`fromNow`](), defaulting to the registered OAuth client `maxExpires` value. |
+
+
+Once the sign in is complete, Taskcluster will
+redirect back to the `redirect_uri` you specified in the URL.
+The redirect URI will have the authorization `code` in the
+query parameter which will then be used to grab a third party access token.
+In the case of an implicit grant (i.e., `response_type=“token”`) the query
+parameter will instead have the access token so you can safely skip the next
+section and continue at [Exchanging a Third Party token for
+Taskcluster Credentials](#exchanging-a-third-party-token-for-taskcluster-credentials) section.
+
+### Exchanging a code for a third-party access token
 
```

Comment:
This might be a good place for a `<Warning>` distinguishing "third party access token" from "(taskcluster) access token".

Reply:
Good idea. I'll add a note. We can make it use the `Warning` component in https://github.com/taskcluster/taskcluster/pull/1255.

Reply:
It's all coming together now :D